src/xmipp/libraries/data/transform_geometry.cpp

```diff
@@ -259,6 +259,14 @@ void ProgTransformGeometry::processImage(const FileName &fnImg,
 		rowOut.setValue(MDL_SHIFT_X, -posp(0));
 		rowOut.setValue(MDL_SHIFT_Y, -posp(1));
 		T.initIdentity(3);
+		int nx;
+		rowIn.getValue(MDL_XCOOR, nx);
```

Comment:
This is undefined behavior in case there's no such label.
You have to either check that the label is present, or at least use some default value of `nx` and `ny`.